res/values/strings.xml

```diff
@@ -690,6 +726,14 @@
     <string name="a11y_delivery_status_invalid">Invalid delivery status</string>
     <string name="a11y_message_context_menu_btn_label">Message actions</string>
     <string name="a11y_background_preview_label">Background preview</string>
+    <string name="a11y_profile_image_label">Your profile image</string>
```

Comment:
consider using just `<string name="pref_profile_photo">Profile picture</string>` instead (we do want to use a11y only where the strings actually differ between normal labels)